src/Mapbender/CoreBundle/Element/Type/TargetElementType.php

```diff
@@ -99,9 +98,12 @@ public function getChoicesQueryBuilder(Options $options)
         } else {
             $elm_ids = array();
             foreach ($options['application']->getElements() as $element_entity) {
-                $class = $element_entity->getClass();
-                if ($class::$ext_api) {
-                    $element_entity->getId();
+                /** @var Element $element_entity */
+                $elementComponentClass = $element_entity->getClass();
+                if (class_exists($elementComponentClass)) {
+                    if ($elementComponentClass::$ext_api) {
+                        $element_entity->getId();
```

Comment:
Would be a  great opportunity to remove snake_case and move to a consistent lower camel case.

Reply:
Agreed, will do.